libs/datamodel/core/src/ast/identifier.rs

```diff
@@ -1,6 +1,6 @@
 use super::*;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
 pub struct Identifier {
     pub name: String,
     pub span: Span,
```

Comment:
I think it's fine if we need it, but it could be risky, since two identifiers with the same name but different spans wouldn't match.

Reply:
I just needed it for the tests for some reason 🤷‍♂